drivers/web-worker-driver/src/test/kotlin/app/cash/sqldelight/drivers/worker/JsWorkerDriverTest.kt

```diff
@@ -60,7 +61,7 @@ class JsWorkerDriverTest {
   }
 
   private fun runTest(block: suspend (SqlDriver) -> Unit) = kotlinx.coroutines.test.runTest {
-    val driver = initAsyncSqlDriver("/worker.sql-wasm.js", schema)
+    val driver = initAsyncSqlDriver(js("""new Worker(new URL("./sqljs.worker.js", import.meta.url))""").unsafeCast<Worker>(), schema)
```

Comment:
Any chance to make this better without casts?

Reply:
I decided to remove this `initAsync...` function and the `withSchema` helper since they aren't really relevant to Web Workers like they were to SqlJs. 

Instead I added a `fromScriptUrl` companion function to construct a worker from just the script URL. There's still a unsafe cast to convert `import.meta.url` to a string, but at least that's hidden from most people's code now.